### PR TITLE
Deep-linking capability on Accordion and Tabs components

### PR DESCRIPTION
**Pull request by jckautzmann**
- refactored existing PRs as specific item ID was introduced for Tabs and Accordion:
https://github.com/adobe/aem-core-wcm-components/pull/946
https://github.com/adobe/aem-core-wcm-components/pull/960

### CODE REVIEW

File: content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/clientlibs/site/js/tabs.js [hunk 1]
```diff
@@ -300,6 +309,28 @@
...
             }
         }
+
+        /**
+         * Returns the tab item index that corresponds to the deep link in the URL fragment
+         *
+         * @private
+         * @returns {Number} the tab item index if it exists, -1 otherwise
+         */
+        function getDeepLinkItemIdx() {
```
Comment (vladbailescu) [inline content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/clientlibs/site/js/tabs.js:319]: This is very similar to `getDeepLinkItem`. Could we create just one helper method for getting the item, that is shared between the two usages?

Comment (jckautzmann) [reply content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/clientlibs/site/js/tabs.js]: @vladbailescu It's been addressed in https://github.com/adobe/aem-core-wcm-components/pull/1011/commits/3422ecd9c7dfc2b21784e2261cfe3636b5e76395

File: content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/clientlibs/site/js/accordion.js [hunk 2]
```diff
@@ -578,7 +557,7 @@
                         if (addedNode.querySelectorAll) {
                             var elementsArray = [].slice.call(addedNode.querySelectorAll(selectors.self));
                             elementsArray.forEach(function(element) {
-                                new Accordion({ element: element, options: readData(element), deepLinkId: deepLinkId });
+                                new Accordion({ element: element, options: readData(element), });
```
Comment (bartoszglow) [inline content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/clientlibs/site/js/accordion.js:560]: please remove unnecessary comma

File: content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/clientlibs/site/js/tabs.js [hunk 3]
```diff
@@ -68,6 +68,8 @@
          * @param {TabsConfig} config The Tabs configuration
          */
         function init(config) {
+            that._config = config;
```
Comment (bartoszglow) [inline content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/clientlibs/site/js/tabs.js:71]: where do you use `that._config`?

Comment (jckautzmann) [reply content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/clientlibs/site/js/tabs.js:71]: it's used in the utils: `getDeepLinkItemIdx`

**Pull request merged**